src/apis/whattomine/algorithm.ts

```diff
@@ -20,14 +20,14 @@ export class Algorithm {
   public static Keccak = new Algorithm(900);
   public static X11 = new Algorithm(15000);
   public static X13 = new Algorithm(4800);
-  public static Scrypt = new Algorithm(115200);
+  public static Scrypt = new Algorithm(500);
   public static "SHA-256" = new Algorithm(14000);
-  public static Quark = new Algorithm(150);
+  public static Quark = new Algorithm(3300);
   public static NIST5 = new Algorithm(57);
-  public static Lyra2RE = new Algorithm(13800);
```

Comment:
13800 seems to be the default. i think you got lyra2rev2 confused with lyra2re - https://whattomine.com/coins/134-cto-lyra2re